playwright/integrations/authenticated-chat.spec.ts

```diff
@@ -1,73 +1,191 @@
+                return request.url().includes(OmnichannelEndpoints.LiveChatAuthSessionInitPath);
+            }),
+            page.waitForResponse(response => {
+                return response.url().includes(OmnichannelEndpoints.LiveChatAuthSessionInitPath);
+            }),
+            await page.evaluate(async ({ omnichannelConfig, authUrl }) => {
+                const {OmnichannelChatSDK_1: OmnichannelChatSDK} = window;
+
+                const payload = {
+                    method: "POST"
+                };
+
+                const response = await fetch(authUrl, payload);
+                const authToken = await response.text();
+
+                const chatSDKConfig = {
+                    getAuthToken: () => authToken
+                };
+
+                const chatSDK = new OmnichannelChatSDK.default(omnichannelConfig, chatSDKConfig);
+
+                await chatSDK.initialize();
+
+                const runtimeContext = {
+                    requestId: chatSDK.requestId,
+                    authToken
+                };
+
+                await chatSDK.startChat();
+
+                await chatSDK.endChat();
+
+                return runtimeContext;
+            }, { omnichannelConfig, authUrl })
+        ]);
+
+        const {requestId, authToken} = runtimeContext;
+        const chatTokenRequestUrl = `${omnichannelConfig.orgUrl}/${OmnichannelEndpoints.LiveChatv2AuthGetChatTokenPath}/${omnichannelConfig.orgId}/${omnichannelConfig.widgetId}/${requestId}?channelId=lcw`;
+        const sessionInitRequestUrl = `${omnichannelConfig.orgUrl}/${OmnichannelEndpoints.LiveChatAuthSessionInitPath}/${omnichannelConfig.orgId}/${omnichannelConfig.widgetId}/${requestId}?channelId=lcw`;
+
+        const chatTokenRequestHeaders = chatTokenRequest.headers();
+        const sessionInitRequestHeaders = sessionInitRequest.headers();
+
+        expect(chatTokenRequest.url() === chatTokenRequestUrl).toBe(true);
+        expect(chatTokenRequestHeaders['authenticatedusertoken']).toBe(authToken);
+        expect(chatTokenResponse.status()).toBe(200);
+        expect(sessionInitRequest.url() === sessionInitRequestUrl).toBe(true);
+        expect(sessionInitRequestHeaders['authenticatedusertoken']).toBe(authToken);
+        expect(sessionInitResponse.status()).toBe(200);
+    });
+
+    test('ChatSDK.startChat() with liveChatContext should not perform session init & validate the live work item details & validate auth map record', async ({ page }) => {
+        await page.goto(testPage);
+
+        const requestUrls = [];
+        page.on('request', (request) => {
+            if (request.url().includes("livechatconnector")) {
+                requestUrls.push(request.url());
+            }
+        });
+
+        const [_, liveWorkItemDetailsRequest, liveWorkItemDetailsResponse, liveChatMapRecordRequest, liveChatMapRecordResponse, liveChatSessionCloseRequest, liveChatSessionCloseResponse, runtimeContext] = await Promise.all([
```

Comment:
No need to validate `session close` since this test is only applicable for `live work item details` & `auth map record`